modules/masterbar/masterbar.php

```diff
@@ -123,6 +123,14 @@ public function add_styles_and_scripts() {
 
 		wp_enqueue_script( 'jetpack-accessible-focus', plugins_url( '_inc/accessible-focus.js', JETPACK__PLUGIN_FILE ), array(), JETPACK__VERSION );
 		wp_enqueue_script( 'a8c_wpcom_masterbar_overrides', $this->wpcom_static_url( '/wp-content/mu-plugins/admin-bar/masterbar-overrides/masterbar.js' ), array( 'jquery' ), JETPACK__VERSION );
+		wp_localize_script( 'a8c_wpcom_masterbar_overrides', 'jetpackTracks', array(
+			'siteId' => Jetpack_Options::get_option( 'id' ),
+			'siteUrl' => get_option( 'siteurl' ),
+		) );
+
+		// so tracks events can be fired from the frontend.
+		wp_enqueue_script( 'jp-tracks', '//stats.wp.com/w.js', array( 'a8c_wpcom_masterbar_overrides' ), gmdate( 'YW' ), true );
+		wp_enqueue_script( 'jp-tracks-functions', plugins_url( '_inc/lib/tracks/tracks-callables.js', JETPACK__PLUGIN_FILE ), array( 'a8c_wpcom_masterbar_overrides' ), JETPACK__VERSION, false );
```

Comment:
One more thing occurred to me - shouldn't the dependencies be the other way around? Since we are using the tracks code in `masterbar.js`, should we instead add `jp-tracks` and `jp-tracks-functions` as dependencies for `a8c_wpcom_masterbar_overrides`?